sample_dataset/metabase/sample_dataset/generate.clj

```diff
@@ -60,18 +61,27 @@
      :latitude   (random-latitude)
      :longitude  (random-longitude)
      :source     (rand-nth ["Google" "Twitter" "Facebook" "Organic" "Affiliate"])
-     :created_at (random-date-between (u/relative-date :year -1) (u/relative-date :year 1))}))
+     :created_at (random-date-between (u/relative-date :year -2) (u/relative-date :year 1))}))
 
 ;;; ## PRODUCTS
 
 (defn- random-company-name []
   (first (company/names)))
 
+(defn- rejection-sample
+  [min max dist]
+  (let [x (dist/sample dist)]
+    (if (<= min x max)
+      x
+      (rejection-sample min max dist))))
+
 (defn- random-price [min max]
-  (let [range (- max min)]
-    (-> (rand-int (* range 100))
-        (/ 100.0)
-        (+ min))))
+  (let [range    (- max min)
+        m1       (+ min (* range 0.3))
+        m2       (+ min (* range 0.75))
```

Comment:
What's an `m1` and `m2`? A more verbose name or a comment explaining what these are would make this code a lot easier to understand 

https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#prefer-more-verbose-names

Reply:
+1